### PR TITLE
Update README.md for Sentinels

### PR DESCRIPTION
**Pull request by balapriyac**
- [x] Rephrase content as needed
- [x] Fix typos and formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-# Sentinels 🤖
+# 🤖 Sentinels 
+👋 Welcome to sentinels!
```
Comment (ramesht007) [inline README.md]: Sentinals are the bots themselves. So we thought this name from THE MATRIX movie.
you can check about [sentinals](https://www.google.com/search?q=sentinel+matrix&sxsrf=APq-WBuZLdoN4uvBHRGpPGo7BhdwQMWOXg%3A1643538878522&ei=vmn2YcKaH_2SseMP-8qgoAQ&ved=0ahUKEwjC8Zy4o9n1AhV9SWwGHXslCEQQ4dUDCA4&uact=5&oq=sentinel+matrix&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEJECMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgUIABCABDIFCAAQgAQyBQgAEIAEOgcIIxCwAxAnOgcIABBHELADOgcIABCwAxBDOgoIABDkAhCwAxgAOgwILhDIAxCwAxBDGAFKBAhBGABKBAhGGAFQ2wJY2wJg7AVoAXACeACAAVKIAVKSAQExmAEAoAEByAERwAEB2gEGCAAQARgJ2gEGCAEQARgI&sclient=gws-wiz) here.

`Welcome to Sentinals` is not sounding perfect.
Maybe we can improve that.

**Review by ramesht007 — COMMENTED** [review]
Thanks for the changes!!
We Can work on the welcome message. Rest looks good!!

File: README.md (hunk 1 above)
Comment (balapriyac) [reply README.md]: Yeah, how about say hello to sentinels, or make friends with sentinels?

**Comment by balapriyac** [comment]
@ramesht007 see if the most recent edit works.

**Review by ramesht007 — APPROVED** [review]
Woww Perfect!!

**Pull request merged**